### DISCUSSION TITLE
and in general the click is YES or NO cookies?

### DISCUSSION THREAD

**Discussion by sk2eu** [OhMyGuus/I-Still-Dont-Care-About-Cookies#11212] [General]
HI 

the fact that you can click away these banners is a blessing if you know the free internet from the past...

and i was convinced for many months now that this addon is still NO click for me! because it is data theft and not if a site works. but i guess it is not so... you confirm it with YES ?

why doesn't it all just click NO? what's stopping it? what would you have to do to make this addon click NO to everything? (my data should not be shared and sold)

**Comment by OhMyGuus** [comment]
Currently the point of the extension is to get rid of the cookie notice without breaking the site. 
This means that on certain sites it's denied or hidden and on others it accepts the necessary cookies to get the site working without issues. á

So sometimes it accepts some cookies, however we try to deny as much as possible. Can you give me the site so I can check why it is accepting the cookies over there?

**Comment by sk2eu** [comment]
I can't give you an exact page because 98% almost only steal data. Google had a note at the beginning that they store the data for 20 years, today it's limited to a year or so but... the point here is that these data thieves win the scoring and they don't delete it! They don't care about our data and spit it away like old chewing gum ... they keep the scoring for themselves ... and make profiles of people/children who don't even understand what they click on ... 


Say something about that? DE / PL as an example ... in quite a few of them there are 100 companies that sell the data and with all of them I am supposed to set a tick to NO which is impossible manually...
please make sure that this tool always clicks no! and make it public for schools to protect the young. A whole generation will thank you once they have understood what you have protected them from... now we are just lazy and click YES ... it must be set to NO.  

www.heise.de/
www.welt.de
www.computerbild.de


www.wp.pl/
www.onet.pl
www.oko.press

**Comment by KoljaL** [comment]
Why not click "YES to ALL" and delete everything after leaving the page?

Of cause there have to be an exception maker for services (logins, etc) I want to use.
